### ISSUE TITLE
qt does not link to conda libs (libjpeg, libtiff)

### ISSUE THREAD

**Issue by jschueller** [conda-forge/qt-feedstock#147]
qt imageformats plugins do not link
- to libjpeg as qt wants libjpeg-turbo (it checks for jpeg_crop_scanline, needed for chromium 3rd party)
the bad news is that conda forge wont switch anytime soon: 
https://github.com/conda-forge/conda-forge.github.io/issues/673
- to libtiff (depends on libjpeg), worse it links to ImageIO's framework libTIFF.dylib

**Comment by hmaarrfk** [comment]
i don't think we are against moving to turbo, but we are just slow since it seems involved. your help would be greatly apprecaited moving to turbo.

**Comment by traversaro** [comment]
>     * to libjpeg as qt wants libjpeg-turbo (it checks for jpeg_crop_scanline, needed for chromium 3rd party)
>       the bad news is that conda forge wont switch anytime soon:
>       [conda-forge/conda-forge.github.io#673](https://github.com/conda-forge/conda-forge.github.io/issues/673)

Even if conda-forge does not globally switch to jpeg-turbo, I think it can work fine if any port that requires it just depends on jpeg-turbo .

**Comment by isuruf** [comment]
> Even if conda-forge does not globally switch to jpeg-turbo, I think it can work fine if any port that requires it just depends on jpeg-turbo .

No. We can either switch to jpeg-turbo or not at all. Nothing in the middle please.

**Comment by traversaro** [comment]
> 
> 
> > Even if conda-forge does not globally switch to jpeg-turbo, I think it can work fine if any port that requires it just depends on jpeg-turbo .
> 
> No. We can either switch to jpeg-turbo or not at all. Nothing in the middle please.

Sorry, I meant that I recalled that there are some port already depended on jpeg-turbo, see https://github.com/search?p=4&q=org%3Aconda-forge+libjpeg-turbo&type=Code . If this is not support/discouraged, probably it should be documented somewhere?

**Comment by isuruf** [comment]
> If this is not support/discouraged

It is not.

> probably it should be documented somewhere?

We are all volunteers here. PRs to docs are welcome.

**Comment by traversaro** [comment]
> > probably it should be documented somewhere?
> 
> We are all volunteers here. PRs to docs are welcome.

Sure! I just explained why I had no idea depending on `libjpeg-turbo` was not supported and discouraged, not complaining that it was not documented anywhere.

**Comment by isuruf** [comment]
`libjepg` and `libjpeg-turbo` both have the common symbols and loading both into the same process can be problematic.
This is the same situation with MKL and openblas where we also discourage having both in the same env.

**Comment by hmaarrfk** [comment]
I remember one thing that was brought up was that qt isn't easy to build. With the speed at which all the dependencies are released on conda forge, it would be a large maintenance burden to depend on too many things.
